rotkehlchen/tests/api/test_counterparty_asset_mappings.py

```diff
@@ -0,0 +1,279 @@
+from http import HTTPStatus
+from typing import TYPE_CHECKING, Any, Final
+
+import requests
+
+from rotkehlchen.db.filtering import CounterpartyAssetMappingsFilterQuery
+from rotkehlchen.tests.utils.api import (
+    api_url_for,
+    assert_error_response,
+    assert_proper_sync_response_with_result,
+)
+
+if TYPE_CHECKING:
+    from rotkehlchen.api.server import APIServer
+    from rotkehlchen.globaldb.handler import GlobalDBHandler
+
+NUM_PACKAGED_COUNTERPARTY_ASSETS_MAPPINGS: Final = 0
+
+
+def _get_all_counterparty_mappings(globaldb: 'GlobalDBHandler') -> dict[str, Any]:
+    """Utility function to return all the counterparty asset mappings in the DB."""
+    mappings, mappings_found, mappings_total = globaldb.query_asset_mappings_by_type(
+        mapping_type='counterparty',
+        filter_query=CounterpartyAssetMappingsFilterQuery.make(offset=0, limit=5000),
+    )
+    return {
+        'entries': mappings,
+        'entries_found': mappings_found,
+        'entries_total': mappings_total,
+    }
+
+
+def test_counterparty_asset_mappings_query(
+        rotkehlchen_api_server: 'APIServer',
+        globaldb: 'GlobalDBHandler',
+) -> None:
+    # query all the mappings
+    result = _get_all_counterparty_mappings(globaldb)
+    assert len(result['entries']) == result['entries_found'] == result['entries_total'] == NUM_PACKAGED_COUNTERPARTY_ASSETS_MAPPINGS  # noqa: E501
+
+    # query all common mappings
+    response = requests.post(
```

Comment:
Please don't waste so much space with the comments
```suggestion
    response = requests.post(  # query all common mappings
```